src/DatatypeBindings.cpp

```diff
@@ -284,6 +286,117 @@ void DatatypeBindings::bind(pybind11::module& m){
+        .def(py::init<>())
+        .def_readwrite("i", &IMUReportRotationVectorWAcc::i)
+        .def_readwrite("j", &IMUReportRotationVectorWAcc::j)
+        .def_readwrite("k", &IMUReportRotationVectorWAcc::k)
+        .def_readwrite("real", &IMUReportRotationVectorWAcc::real)
+        .def_readwrite("accuracy", &IMUReportRotationVectorWAcc::accuracy)
+        ;
+
+    py::class_<IMUReportRotationVector, IMUReport, std::shared_ptr<IMUReportRotationVector>>(m, "IMUReportRotationVector", DOC(dai, IMUReportRotationVector))
+        .def(py::init<>())
+        .def_readwrite("i", &IMUReportRotationVector::i)
+        .def_readwrite("j", &IMUReportRotationVector::j)
+        .def_readwrite("k", &IMUReportRotationVector::k)
+        .def_readwrite("real", &IMUReportRotationVector::real)
+        ;
+
+    py::class_<IMUReportGyroIntegratedRV, IMUReport, std::shared_ptr<IMUReportGyroIntegratedRV>>(m, "IMUReportGyroIntegratedRV", DOC(dai, IMUReportGyroIntegratedRV))
+        .def(py::init<>())
+        .def_readwrite("i", &IMUReportGyroIntegratedRV::i)
+        .def_readwrite("j", &IMUReportGyroIntegratedRV::j)
+        .def_readwrite("k", &IMUReportGyroIntegratedRV::k)
+        .def_readwrite("real", &IMUReportGyroIntegratedRV::real)
+        .def_readwrite("angVelX", &IMUReportGyroIntegratedRV::angVelX)
+        .def_readwrite("angVelY", &IMUReportGyroIntegratedRV::angVelY)
+        .def_readwrite("angVelZ", &IMUReportGyroIntegratedRV::angVelZ)
+        ;
+
+    py::class_<IMUDatas> imuDatas(m, "IMUDatas", DOC(dai, IMUDatas));
+    imuDatas
+        .def(py::init<>())
+        .def_readwrite("acceleroMeter", &IMUDatas::acceleroMeter)
+        .def_readwrite("linearAcceleroMeter", &IMUDatas::linearAcceleroMeter)
+        .def_readwrite("gravity", &IMUDatas::gravity)
+        .def_readwrite("gyroscope", &IMUDatas::gyroscope)
+        .def_readwrite("gyroscopeUncalibrated", &IMUDatas::gyroscopeUncalibrated)
+        .def_readwrite("magneticField", &IMUDatas::magneticField)
+        .def_readwrite("magneticFieldUncalibrated", &IMUDatas::magneticFieldUncalibrated)
+        .def_readwrite("rotationVector", &IMUDatas::rotationVector)
+        .def_readwrite("gameRotationVector", &IMUDatas::gameRotationVector)
+        .def_readwrite("geoMagRotationVector", &IMUDatas::geoMagRotationVector)
+        .def_readwrite("arvrStabilizedRotationVector", &IMUDatas::arvrStabilizedRotationVector)
+        .def_readwrite("arvrStabilizedGameRotationVector", &IMUDatas::arvrStabilizedGameRotationVector)
+        .def_readwrite("gyroIntegratedRotationVector", &IMUDatas::gyroIntegratedRotationVector)
+        ;
+
+
+    // Bind RawIMUData
+    py::class_<RawIMUData, RawBuffer, std::shared_ptr<RawIMUData>> rawIMUDatas(m, "RawIMUData", DOC(dai, RawIMUData));
+    rawIMUDatas
+        .def_readwrite("imuDatas", &RawIMUData::imuDatas)
+        ;
```

Comment:
Missing bindings

Reply:
Do you guys have plans to support any other IMU's ? It would be nice to know the supported IMU IC's out-of-the box by the DepthAI SoM for custom carrier boards.

We are not aware of this and we added an external IMU on our carrier board which is interfaced to the compute SoM. IMU interfaced directly to DepthAI SoM will be great rather than interfacing it via external compute SoM.

Reply:
So the BNO080/BNO085/BNO086 are all supported by DepthAI.  The OAK-D, OAK-D-WIFI, OAK-D-POE, and DM1092 all have the BNO085 onboard.  We paid a license fee so we can field-uprade any BNO080/BNO085 to BNO086 as well.
So the BNO085 can actually be upgraded to BNO086 if you'd like.  We do not have any plans at this time for other IMU support.